Lib/test/test_exceptions.py

```diff
@@ -318,11 +319,17 @@ def baz():
         check('(yield i) = 2', 1, 2)
         check('def f(*):\n  pass', 1, 7)
 
+    @unittest.skipIf(ctypes.sizeof(ctypes.c_ssize_t) > ctypes.sizeof(ctypes.c_int),
+                     "Py_ssize_t is bigger than INT_MAX, so this is unreachable")
```

Comment:
This should probably the other way around. If `sizeof(ssize_t) <= sizeof(int)` the cast is always safe, right?

Reply:
Yeah :upside_down_face: